es6-promises/main.js

```diff
@@ -0,0 +1,10 @@
+import takeAChance from './take-a-chance.js';
+
+var chance = takeAChance('Nathan');
+
+chance.catch(error => {
```

Comment:
I would have the catch statement at the end of any `.then` s as it is the more standard place.